src/frontend/src/views/Terms/index.tsx

```diff
@@ -195,31 +182,7 @@ export default function Terms(): JSX.Element {
               items={[
                 <span key="0">
                   <ListItemLabel>Within your organization:</ListItemLabel>
-                  The pathogen genomes created from your Samples and the
-                  analytical outputs you create using Aspen are visible to other
-                  Users at your organization (ex: your DPH). You, along with
-                  other members of your Group, control whether you permit us to
-                  share this information with Users outside your organization,
-                  including as discussed below for public repositories like
-                  GISAID. In certain circumstances, we may also share your data
-                  with third party entities such as CDPH, through the Aspen
-                  tool, in line with your organization’s policies or in line
-                  with applicable law.
-                </span>,
-                <span key="1">
-                  <ListItemLabel>With public repositories:</ListItemLabel>
-                  Because of the time-sensitive nature of genomic pathogen data,
-                  we strongly encourage you to share de-identified pathogen
-                  genome sequences with the broader scientific and public health
-                  community as soon as possible.{" "}
-                  <B>
-                    For this reason, unless you choose to mark a Sample as
-                    ‘private’ within 2 weeks of upload, we will automatically
-                    share Pathogen Consensus Genomes with public repositories.
-                  </B>{" "}
-                  However, we recognize that this is not always your preference.
-                  Thus, you may choose to label your data as Private, so that it
-                  cannot be seen by anyone outside of your group.
+		  The pathogen genomes created from your Samples and the analytical outputs you create using Aspen are visible to other Users at your organization (ex: your DPH). You, along with other members of your Group, control whether you permit us to share this information with Users outside your organization. In certain circumstances, we may also share your data with third party entities, through the Aspen tool, in line with your organization’s policies or in line with applicable law.  Samples marked &quot;private&quot; will never be shared with any 3rd parties unless you choose to mark them &quot;public&quot; later on.
```

Comment:
I'd like to have this get some newlines just to make it easier to read at a glance in code editor